scripts/setup.sh

```diff
@@ -0,0 +1,109 @@
+# !/usr/bin/env bash
+# Creates the folder structure defined in folder structure section below
+function createFolderStructure() {
+     depth="1"
+     while (( "$#" )); do
+         while (( $1 != $depth )); do
+             cd ..
+             (( depth-- ))
+         done
+         shift
+         mkdir "$1"
+         cd "$1" || exit
+         (( depth++ ))
+         shift
+       done
+     while (( 1 != $depth )); do
+         cd ..
+         (( depth-- ))
+     done
+}
+
+# Step-1 | Create Folder Strcuture
+read -r -d '' FOLDERSTRUCTURE << EOM
+1 QBFT-Network
+    2 Node-1
+        3 data
+    2 Node-2
+        3 data
+    2 Node-3
+        3 data
+    2 Node-4
+        3 data
+EOM
+
+
+if [ ! -d "./QBFT-Network" ]
+then
+    : # do nothing
+else
+    sudo rm -r -f ./QBFT-Network
+fi
+
+createFolderStructure $FOLDERSTRUCTURE
+
+# Step-2 | Create a QBFT genesis file
+function createQBFTConfigFile() {
```

Comment:
Let's make this script generic, so it can be used for other consensus as well.

Reply:
I removed the script, will work on different option to do the testing.